docs/projects/import-export/_posts/2019-10-01-javascript.md

```diff
@@ -0,0 +1,57 @@
+---
+title: importing and exporting a variable in JavaScript  
+layout: default  
+last-modified: 2019-10-01  
```

Comment:
You can probably update the date here (as well as in the title). Same goes for the other articles.

Reply:
Fixed dates!